src/Services/Message/index.js

```diff
@@ -15,9 +16,26 @@ const createSystemMessage = (text) => {
   return message;
 };
 
+const addMessageToMessages = async (id, text, user) => {
```

Comment:
yapılan iş dolayısıyla mesajı sadece oda içine eklemekle kalmayıp redis'e de kaydettiği için bence şöyle bir isimlendirme daha olabilir `saveMessage`.